src/core/account/account-init.ts

```diff
@@ -68,6 +71,9 @@ export async function ensureAccountExists(
   loginTree: LoginTree,
   appId: string
 ): Promise<LoginTree> {
+  // For crash errors:
+  ai.props.log.breadcrumb('ensureAccountExists', {})
```

Comment:
Do you mean to move `ai.props.log.breadcrumb('ensureAccountExists createChildLogin', {})` to line 82 and `ai.props.log.breadcrumb('ensureAccountExists make repo', {})` to line 87?

Reply:
More like this:

```ts
  // If there is no app login, make that:
  const login = searchTree(loginTree, login => login.appId === appId)
  if (login == null) {
    ai.props.log.breadcrumb('createChildLogin', {})
    return await createChildLogin(ai, loginTree, loginTree, appId)
  }

  // Otherwise, make the repo:
  if (findFirstKey(login.keyInfos, accountType) == null) {
    ai.props.log.breadcrumb('createAccountRepo', {})
    ...
  }
```